api/ai/priority_algorithm/priority_algorithm.py

```diff
@@ -130,6 +111,18 @@ def _unpack_priority_team_set(self, priority_team_set: PriorityTeamSet) -> TeamS
 
         return TeamSet(teams=teams)
 
+    def mutate(
+        self,
+        team_set: PriorityTeamSet,
+    ) -> List[PriorityTeamSet]:
+        return [
+            mutated_set
+            for mutation in self.algorithm_config.MUTATIONS
+            for mutated_set in mutation.mutate(
```

Comment:
the ordering of the 2 `for` parts seems backwards for this list comprehension to work, maybe a quick sanity test that it always returns a list of PriorityTeamSet and not a list of lists or something?

Reply:
I added a test for it :)